refinery/config/settings/dev.py

```diff
@@ -7,6 +7,6 @@
 
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
 
-INSTALLED_APPS += (
+INSTALLED_APPS += (  # NOQA: F405
```

Comment:
This change must be due to the fact that this file hasn't been edited in ages?

Reply:
Yes. I edited it before I refactored into base.